src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/PackageAnnotationCheck.java

```diff
@@ -48,6 +48,19 @@
  * <pre>
  * &lt;module name=&quot;PackageAnnotation&quot;/&gt;
  * </pre>
+ * <p>
+ * Examples of validating files package-info.java and MyClass.java:
+ * </p>
+ * <p>package-info.java</p>
+ * <pre>
+ * &#64;Deprecated
+ * package com.example.annotations.packageannotation; //ok
+ * </pre>
+ * <p>MyClass.java</p>
+ * <pre>
+ * &#64;Deprecated
+ * package com.example.annotations.packageannotation; //violation
```

Comment:
Please make violation case first and then show good case. This makes it look like we are changing bad code to good.

Reply:
Done